core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -203,7 +204,22 @@ func (i *pluginOracleCreator) Create(donID uint32, config cctypes.OCR3ConfigWith
 	if err != nil {
 		return nil, err
 	}
-	return oracle, nil
+
+	closeFunc := func() error {
+		errs := make([]error, 0)
+
+		for _, cr := range contractReaders {
+			errs = append(errs, cr.Close())
+		}
+
+		for _, cw := range chainWriters {
+			errs = append(errs, cw.Close())
+		}
+
+		return multierr.Combine(errs...)
```

Comment:
We're trying to move away from `multierr`. How about using the std lib?
```suggestion
	closeFunc := func() (err error) {
		for _, cr := range contractReaders {
			err = errors.Join(err, cr.Close())
		}

		for _, cw := range chainWriters {
			err = errors.Join(err, cw.Close())
		}

		return
```
Or, you can close these in parallel with via helpers:
```suggestion
	closeFunc := func() error {
		return services.CloseAll(slices.Concat[io.Closer](contractReaders, chainWriters)...)
```